wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <v-card class="my-7 mx-auto w-75">
+        <v-sheet color="secondary">
+            <v-card-title>Leaderboard</v-card-title>
+        </v-sheet>
+        <v-table class="mt-7 mx-auto w-75">
+            <thead>
+                <tr>
+                    <th class="text-h6 text-center Name"><strong>Player Name</strong></th>
+                    <th class="text-h6 text-center GameCount">
+                        <strong>Game Count</strong>
+                    </th>
+                    <th class="text-center text-h6 AverageAttempts">
+                        <strong>Average Guesses</strong>
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="player in topTenScores" :key="player.name">
+                    <td class="text-center">{{ player.name }}</td>
+                    <td class="text-center">{{ player.gameCount }}</td>
+                    <td class="text-center">{{ player.averageAttempts }}</td>
```

Comment:
since you have the optional parameter for time you could implement the extra credit by `{{player.AverageSecondsPerGame}}`.